lighthouse-core/audits/font-display.js

```diff
@@ -25,11 +25,13 @@ const UIStrings = {
     'webfonts are loading. ' +
     '[Learn more](https://web.dev/font-display/).',
   /**
-   * @description A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts and that the user will need to manually check it.
-   * @example {https://font.cdn.com/} fontURL
+   * @description [ICU Syntax] A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts, telling the user that they will need to manually check them.
+   * @example {https://font.cdn.com/} fontOrigin
    */
-  undeclaredFontURLWarning: 'Lighthouse was unable to automatically check the font-display value ' +
-    'for the following URL: {fontURL}.',
+  undeclaredFontOriginWarning:
+    'Lighthouse was unable to automatically check the {fontCountForOrigin, plural, ' +
+    '=1 {`font-display` value for the following origin: {fontOrigin}.} ' +
+    'other {`font-display` values for the following origin: {fontOrigin}.}}',
```

Comment:
fyi @exterkamp here's a fun new nesting variant we'll be trying out. #9460 _should_ handle direct replacement (and a backticked phrase!) inside a plural, but I guess we'll see :)

Reply:
Fun! New i18n cases 😄  It seems like it should work to me 🤷 

Do you have a test LHR/screenshot with it rendered correctly? The usual vercel deploys don't afaict?

Reply:
> Do you have a test LHR/screenshot with it rendered correctly? The usual vercel deploys don't afaict?

yeah, expand the before/after section of the PR description for screenshots

Reply:
You're getting too fancy with these PR descriptions ;)